### ISSUE TITLE
is lgsvl team dismissed in the year?

### ISSUE THREAD

**Issue by jackxia123** [lgsvl/simulator#2043]
is lgsvl team dismissed in the year?, i just want touse your product, nobody will maintain this now, right?

**Comment by JordanCrawfordx** [comment]
I've been wondering this myself. It's a shame the project's being shut down. Such a cool piece of software.

**Comment by EricBoiseLGSVL** [comment]
Yes, the project has been sunset.  Thanks to everyone for making svl simulator such a great project especially those that reported bugs and helped others solve issues.  It has been a pleasure helping and watching everyone leverage the simulator for so many cool projects. One user has continued development here #1994.
